dr-elephant/dr-elephant.sh

```diff
@@ -192,24 +186,21 @@ Therefore, fetcher does not use other REST calls, which may have significant mem
 </fetchers>
 EOF
   # Enable compress for making metrics accessible by dr elephant
-  echo "spark.eventLog.compress = true" >> $SPARK_CONF_DIR/spark-defaults.conf
+  echo "spark.eventLog.compress = true" >>"${SPARK_CONF_DIR}/spark-defaults.conf"
 }
 
-
-function prepare_mysql(){
+function prepare_mysql() {
   systemctl restart mysql
-  mysql -u root -proot-password -e " \
-  CREATE DATABASE drelephant;"
+  mysql -u root -proot-password -e "CREATE DATABASE drelephant;"
```

Comment:
It will be nice to have an option to use Cloud SQL [via Cloud SQL Proxy]

Can we work on that as well, if possible..